### PR TITLE
feature/Adding Brazilian Portuguese translations

### PR DESCRIPTION
**Pull request by niugait**
### Changes

Added Brazilian Portuguese translations

* Updated from latest version (2019-09-26)

### CODE REVIEW

**Comment by joshcanhelp** [comment]
@niugait - Thank you for doing this (again)! I'll submit this to one of the engineers on our team that speaks Portuguese for review.

**Comment by niugait** [comment]
@joshcanhelp No problem, let me know if you need anything else.

**Comment by joshcanhelp** [comment]
@niugait - Can you apply all the suggestions from Luis here? I tried to commit them but it's telling me I don't have the permissions to do so. Thank you!

File: languages/wp-auth0-pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,1965 @@
...
+msgid "This can be fixed one of 2 ways:"
+msgstr "Isso pode ser corrigido de duas maneiras:"
+
+#: lib/WP_Auth0_DBManager.php:242
+msgid "Create a new Management API token"
+msgstr "Crie um novo token da API de Gerenciamento"
+
+#: lib/WP_Auth0_DBManager.php:245
+msgid "and save it in the Auth0 > Settings > Basic tab > API Token field."
+msgstr "e salve-o na guia Auth0 > Configurações > Básico > campo Token da API."
```
[the remark this answers is not in the document]
Comment (niugait) [reply languages/wp-auth0-pt_BR.po]: Shouldn't we also translate the tabs 'Settings and Basic'?

File: languages/wp-auth0-pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,1965 @@
...
+msgstr "Esta informação é encontrada abaixo do campo Client Secret no "
+
+#: lib/admin/WP_Auth0_Admin_Basic.php:201
+msgid "This value can be found the Application settings in the "
+msgstr "Este valor pode ser encontrado nas Configurações do Aplicativo em "
+
+#: lib/admin/WP_Auth0_Admin_Basic.php:203
+msgid " under Show Advanced Settings > OAuth > \"JsonWebToken Signature Algorithm\""
+msgstr ""
+" em Mostrar Configurações Avançadas > OAuth > \"Algoritmo de Assinatura JsonWebToken\""
```
[the remark this answers is not in the document]
Comment (niugait) [reply languages/wp-auth0-pt_BR.po]: Same question here, why shouldn't we translate the tabs?

File: languages/wp-auth0-pt_BR.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshcanhelp) [reply languages/wp-auth0-pt_BR.po]: @niugait - God catch. I think Luis assumed these were labels in the Auth0 Dashboard, not in the WP plugin. The plugin translates those so I believe you're right.

**Comment by niugait** [comment]
> 
> 
> @niugait - Can you apply all the suggestions from Luis here? I tried to commit them but it's telling me I don't have the permissions to do so. Thank you!

Done!

**Comment by niugait** [comment]
@joshcanhelp Done!

**Comment by joshcanhelp** [comment]
Formatting checks find no errors and tests pass:

```
Runtime:       PHP 7.2.18 with Xdebug 2.6.0
Configuration: /Users/josh-cunningham/Sites/wp-auth0/wp-content/plugins/auth0/phpunit.xml.dist

...............................................................  63 / 340 ( 18%)
............................................................... 126 / 340 ( 37%)
............................................................... 189 / 340 ( 55%)
............................................................... 252 / 340 ( 74%)
............................................................... 315 / 340 ( 92%)
.........................                                       340 / 340 (100%)

Time: 1.29 minutes, Memory: 54.00MB

OK (340 tests, 1384 assertions)
```

**Comment by joshcanhelp** [comment]
*Muito obrigado* @niugait!

**Comment by niugait** [comment]
You're welcome! If you guys need some extra tweaks on this, just let me know.

**Comment by niugait** [comment]
You're welcome @joshcanhelp ! If you guys need some extra tweaks on this, just let me know.

**Pull request merged**